packages/neon-core/src/u/Fixed8.ts

```diff
@@ -3,24 +3,29 @@ import { reverseHex } from "./misc";
 
 const DECIMALS = 100000000;
 
-// 0x7fffffffffffffff = 9223372036854775807 (maximum hex fixed8)
-const MAX_FIXED8_HEX = new BN(2).pow(63).minus(1);
-export const MAX_FIXED8 = MAX_FIXED8_HEX.div(DECIMALS);
-
-// 0x8000000000000000 = -9223372036854775808 (minimum hex fixed8)
-const MIN_FIXED8_HEX = new BN(2).pow(63).negated();
-export const MIN_FIXED8 = MIN_FIXED8_HEX.div(DECIMALS);
-
-// Total hex fixed8 number. This inludes positve and negative numbers.
-// 0xffffffffffffffff = 18446744073709551615
-const TOTAL_FIXED8_HEX = new BN(2).pow(64);
 /**
  * A fixed point notation used widely in the NEO system for representing decimals.
  * It is basically a hexideciaml integer that is divided by the 10^8.
  * Supports up to 8 decimals and is 8 bytes long.
  * @extends BN
  */
 export class Fixed8 extends BN {
+  // The maximum Fixed8 is obtained by dividing 0x7fffffffffffffff (= 9223372036854775807) with 10^8.
+  public static MAX_VALUE = new BN(2)
+    .pow(63)
+    .minus(1)
+    .div(DECIMALS);
+
+  // The minimum Fixed8 is obtained by dividing 0x8000000000000000 (= -9223372036854775808) with 10^8.
+  public static MIN_VALUE = new BN(2)
+    .pow(63)
+    .negated()
+    .div(DECIMALS);
+
+  // TOTAL represents the total number of Fixed8 available. This inludes positve and negative fixed8 numbers.
+  // 2^64 = 0xffffffffffffffff = 18446744073709551615
+  public static TOTAL = new BN(2).pow(64);

```

Comment:
not sure if i want to expose this. What is the usecase?

Reply:
You right. I will not expose it.